eras/babbage/impl/src/Cardano/Ledger/Babbage/TxOut.hs

```diff
@@ -422,23 +419,23 @@ instance
   ( Era era
   , Val (Value era)
   , FromCBOR (Annotator (Script era))
-  , DecodeNonNegative (Value era)
+  , FromCBOR (Value era)
   ) =>
   FromCBOR (BabbageTxOut era)
   where
-  fromCBOR = decodeBabbageTxOut
+  fromCBOR = fromCBOR
 
 instance
   ( Era era
   , Val (Value era)
   , FromCBOR (Annotator (Script era))
-  , DecodeNonNegative (Value era)
+  , FromCBOR (Value era)
   ) =>
   FromSharedCBOR (BabbageTxOut era)
   where
   type Share (BabbageTxOut era) = Interns (Credential 'Staking (EraCrypto era))
   fromSharedCBOR credsInterns =
-    internTxOut <$!> decodeBabbageTxOut
+    internTxOut <$!> fromCBOR
```

Comment:
Why this change?
```suggestion
    internTxOut <$!> decodeBabbageTxOut
```

Reply:
Oh no! This is from the `rebase`. Sorry! I hadn't reviewed all the changes yet. I'm doing so now and keeping this ready for another review.